invoices/templates/snippets/weekly_report.html

```diff
@@ -0,0 +1,125 @@
+{% load humanize %}
```

Comment:
These seem to be almost exact copies of per-invoice report pages. If that is the case, please split common part to a separate file and include it.

Reply:
The common part is now included in "snippets/project_hours.html"